templates/www.conf.j2

```diff
@@ -408,4 +410,8 @@ pm.max_requests = {{ php_fpm_max_requests }}
 ;php_flag[display_errors] = off
 ;php_admin_value[error_log] = /var/log/fpm-php.www.log
 ;php_admin_flag[log_errors] = on
-;php_admin_value[memory_limit] = 32M
\ No newline at end of file
+;php_admin_value[memory_limit] = 32M
+
+{% if php_fpm_display_errors is defined and php_fpm_display_errors == 'On' %}
```

Comment:
Whats the reasoning here behind this single rule?

Reply:
I think we should have a way of inserting custom rules into this pool file, much like we do with the ini files.

If you really want to plan ahead, we could also use a way of creating multiple fpm pools, as this is hardcoded to only create one; but it's not blocking or urgent.